Javascript/README.md

```diff
@@ -0,0 +1,84 @@
+# Bitcamp JavaScript :rocket: 
+
```

Comment:
We actually recently updated the README.md format. It's pretty similar, but you will just have to move some stuff around: https://github.com/bitprj/BitCamp/tree/master/repo_logistics/sample-camp